### PR TITLE
Platform release notes for 799xx

### PR DESCRIPTION
**Pull request by deepakbolangady**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/platform.md [hunk 1]
```diff
@@ -15,37 +15,77 @@ import delete_project from './static/delete-project.png'
...
+  Now, for accounts that are only on Harness NextGen, Harness sends password reset emails from Harness NextGen.
+
+- Event notifications were not sent for Pipeline Start events after Harness Platform was upgraded to version 79606. The issue was specific to remote pipelines. (CDS-73805)
+  
+  This issue is now fixed. 
+
+- You could not create Azure Key Vault connectors in Harness NextGen even when you used the service principal credentials that successfully created Azure Key Vault connectors in Harness FirstGen. After you entered the service principal credentials, the Vault setup window stops responding. After several minutes, the following message is displayed: None of the active delegates were available to complete the task. ==> : 'Missing capabilities: [https:null.vault.azure.net]' (PL-39783, ZD-46756)
+
+  This issue is now fixed.
+
```
Comment (brian-f-harness) [inline release-notes/platform.md:75]:
```suggestion
This item requires Harness Delegate version 79xxx. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).

```

File: release-notes/platform.md [hunk 2]
```diff
@@ -15,37 +15,77 @@ import delete_project from './static/delete-project.png'
...
+- Previously, regardless of whether your account was on Harness NextGen or Harness FirstGen, Harness sent password reset emails from Harness FirstGen. This approach failed for accounts that are only on Harness NextGen. (PL-38735)
 
-  A code enhancement has fixed this issue.
+  Now, for accounts that are only on Harness NextGen, Harness sends password reset emails from Harness NextGen.
+
+- Event notifications were not sent for Pipeline Start events after Harness Platform was upgraded to version 79606. The issue was specific to remote pipelines. (CDS-73805)
+  
+  This issue is now fixed. 
+
+- You could not create Azure Key Vault connectors in Harness NextGen even when you used the service principal credentials that successfully created Azure Key Vault connectors in Harness FirstGen. After you entered the service principal credentials, the Vault setup window stops responding. After several minutes, the following message is displayed: None of the active delegates were available to complete the task. ==> : 'Missing capabilities: [https:null.vault.azure.net]' (PL-39783, ZD-46756)
```
Comment (brian-f-harness) [inline release-notes/platform.md:72]:
```suggestion
- You could not create Azure Key Vault connectors in Harness NextGen even when you used the service principal credentials that successfully created Azure Key Vault connectors in Harness FirstGen. After you entered the service principal credentials, the Vault setup window stopped responding. After several minutes, the following message is displayed: None of the active delegates were available to complete the task. ==> : 'Missing capabilities: [https:null.vault.azure.net]' (PL-39783, ZD-46756)
```

File: release-notes/platform.md [hunk 3]
```diff
@@ -15,37 +15,77 @@ import delete_project from './static/delete-project.png'
...
+
+  This issue is now fixed.
+
+- The AWS connector widget's prefix field did not accept prefixes starting with a slash. Such slashes were stripped off, and this led to undesired behavior. (PL-39194, ZD-45104)
+
+Prefixes that begin with a slash are now supported. 
+
+This item requires Harness Delegate version 79xxx. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
+
+- - Account-level connectors with resource groups set to **Specified** were not available at the project-level. (PL-38828, ZD-44474).
```
Comment (brian-f-harness) [inline release-notes/platform.md:82]:
```suggestion
- Account-level connectors with resource groups set to **Specified** were not available at the project-level. (PL-38828, ZD-44474). 
```

**Review by StrawberryPad — DISMISSED** [review]

File: release-notes/platform.md (hunk 1 above)
Comment (deepakbolangady) [reply release-notes/platform.md:71]: @brian-f-harness , I assume you missed the text below this suggestion.

**Review by brian-f-harness — APPROVED** [review]
LGTM

**Pull request merged**